SPECS/openssl/openssl.spec

```diff
@@ -51,94 +44,18 @@ Patch11:  0011-Remove-EC-curves.patch
 Patch12:  0012-Disable-explicit-ec.patch
 # # Skipped tests from former 0011-Remove-EC-curves.patch
 Patch13:  0013-skipped-tests-EC-curves.patch
-# # Instructions to load legacy provider in openssl.cnf
+# # Instructions to load ly provider in openssl.cnf
```

Comment:
We should review this patch to see if we should keep or remove it. The default still uses secure crypto with this patch applied, but removing this patch also removes a user's ability to easily activate the less secure crypto (unless they know the specific setting so this suggestion is closer to obfuscation)